src/utils/joi/joi.types.ts

```diff
@@ -72,58 +64,8 @@ export const joiContentId = Joi.string().external(
   }
 );
 
-export const joiPoligon = Joi.array()
-  .items(Joi.array().items(Joi.number()))
-  .external(async (value: number[][] | undefined, _helpers: any) => {
-    if (value !== undefined) {
-      const isValid =
-        value.length &&
-        value.every(
-          (coordinateArray: number[]) =>
-            coordinateArray.length === 2 &&
-            coordinateArray.every((coordinate: number) =>
-              coordinateAxisRegex.test(coordinate.toString())
-            )
-        );
-      if (!isValid) {
-        throw new InvalidCoordinateError();
-      }
-      const givenPolygon = turf.polygon([
-        value.map((coordinateArray: number[]) =>
-          coordinateArray.map((coordinate: number) => +coordinate)
-        ),
-      ]);
-      const isIntersecting = (await ItemRPCService.getAreas()).some(
-        (area: IArea) => {
-          const areaPolygon = turf.polygon([area.polygon]);
-          return !!turf.intersect(givenPolygon, areaPolygon);
-        }
-      );
-      if (isIntersecting) {
-        throw new PoligonIntersectionError();
-      }
-    }
-    return value;
-  });
-
-export const joiCoordinate = Joi.array()
```

Comment:
make sure to delete uses of all of those schemas

Reply:
I did.
It will merge with another cr on user-service